scss/lessonCard.scss

```diff
@@ -0,0 +1,21 @@
+.card-title {
+  font-size: 1.5rem;
+  font-weight: bold;
+  line-height: 1.33;
+}
+.icon-messages {
```

Comment:
according to BEM, this should be `.lesson-card__icon-container`

Reply:
do you need all the css properties? What if you just gave each a `mr-5` class name instead?

Reply:
Fixed using comment in chat 👍